abnormal-security/info.json

```diff
@@ -0,0 +1,1259 @@
+          "description": "Specify the email address of the employee whose identity analysis you want to retrieve from the Abnormal Security server."
+        },
+        {
+          "title": "Mock Data",
+          "required": false,
+          "visible": true,
+          "editable": true,
+          "type": "checkbox",
+          "name": "mock-data",
+          "tooltip": "Returns test data if set to True.",
+          "description": "Returns test data if set to True.",
+          "value": false
+        }
+      ],
+      "output_schema": {
+        "data": [
+          {
+            "key": "",
+            "name": "",
+            "description": "",
+            "values": [
+              {
+                "value": "",
+                "percentage": "",
+                "total_count": ""
+              }
+            ]
+          }
+        ]
+      }
+    },
+    {
+      "operation": "get_employee_login_info",
+      "annotation": "get_employee_login_info",
+      "category": "investigation",
+      "title": "Get Employee Login Information",
+      "description": "Get employee login information for last 30 days in csv format.",
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "Email Address",
+          "required": true,
+          "visible": true,
+          "editable": true,
+          "type": "text",
+          "name": "emailAddress",
+          "tooltip": "Specify the email address of the employee whose login information you want to retrieve from the Abnormal Security server.",
+          "description": "Specify the email address of the employee whose login information you want to retrieve from the Abnormal Security server."
+        },
+        {
+          "title": "Mock Data",
+          "required": false,
+          "visible": true,
+          "editable": true,
+          "type": "checkbox",
+          "name": "mock-data",
+          "tooltip": "Returns test data if set to True.",
+          "description": "Returns test data if set to True.",
+          "value": false
+        }
+      ],
+      "output_schema": {}
+    },
+    {
+      "operation": "get_detection_reports",
+      "annotation": "get_detection_reports",
+      "category": "investigation",
+      "title": "Get Detection Reports",
+      "description": "Get a list of Detection 360° reports that you have submitted and view corresponding details for each case, including report summaries, statuses, message analyses, and more.",
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "Inquiry Type",
+          "required": true,
+          "visible": true,
+          "editable": true,
+          "type": "select",
+          "name": "inquiry_type",
+          "tooltip": "Specify whether to fetch missed attacks or false positives.",
+          "description": "Specify whether to fetch missed attacks or false positives.",
+          "options": [
+            "MISSED_ATTACK",
```

Comment:
Make the options more friendly looking for UI.

Reply:
done